crt_portal/cts_forms/phone_regex.py

```diff
@@ -0,0 +1,16 @@
+# This regex is designed with U.S. and international phone numbers in mind.
```

Comment:
We could move this to a test

Reply:
I'd like to keep a code comment near the regex that explains its intention. 

But testing both US and international phone numbers and calling that out in the comments makes sense to me!

Reply:
Ok

Reply:
Oh wait! Maybe I misunderstood what you were saying ... were you suggesting writing tests for each of the PASS/FAIL cases I listed?